splash/qtutils.py

```diff
@@ -141,9 +140,19 @@ def qurl2ascii(url):
     return url
 
 
+def to_qurl(s):
+    if isinstance(s, QUrl):
+        return s
+
+    if isinstance(s, unicode):
+        s = s.encode('utf-8')
+
+    return QUrl.fromEncoded(s)
```

Comment:
[Docs](http://doc.qt.io/qt-5/qurl.html#fromEncoded) say "Parses input and returns the corresponding QUrl. input is assumed to be in encoded form, containing only ASCII characters.", but we're not escaping non-ASCII chars. Is it an issue with Qt documentation, or a feature they don't officially support?